sources/world/country-codes.csv

```diff
@@ -0,0 +1,250 @@
+iso2,iso3,iso_numeric
+AF,AFG,4
```

Comment:
The [ISO numeric codes are three digits](https://en.wikipedia.org/wiki/ISO_3166-1_numeric) so some countries will need leading zeros. 

For example: 
```suggestion
AF,AFG,004
```

Reply:
Yep, sorry about that, thanks for the pointer!